README.md

```diff
@@ -1 +1,163 @@
-# msgraph-sdk-php-core
\ No newline at end of file
+# Get started with the Microsoft Graph SDK for PHP
+
+[![Latest Stable Version](https://poser.pugx.org/microsoft/microsoft-graph-core/version)](https://packagist.org/packages/microsoft/microsoft-graph-core)
+
+## Get started with the PHP Connect Sample
+If you want to play around with the PHP library, you can get up and running quickly with the [PHP Connect Sample](https://github.com/microsoftgraph/php-connect-sample). This sample will start you with a little Laravel project that helps you with registration, authentication, and making a simple call to the service.
+
+## Install the SDK
+You can install the PHP SDK with Composer, either run `composer require microsoft/microsoft-graph-core`, or edit your `composer.json` file:
+```
+{
+    "require": {
+        "microsoft/microsoft-graph": "^0.1.0"
```

Comment:
Wrong package name

Reply:
> @SilasKenneth Have you considered [stackoverflow.com/questions/45764872/how-to-push-an-existing-branch-into-a-new-repo-using-git](https://stackoverflow.com/questions/45764872/how-to-push-an-existing-branch-into-a-new-repo-using-git) ? Pushing the old repo branch seems like a better way

Hi @nikithauc, thanks for the suggestion, tried doing that but still I had to resolve the issue with unrelated histories when creating a PR to main. Didn't want to push directly to main.